install/kubernetes/helm/Istio/templates/istio-ingress.yaml

```diff
@@ -0,0 +1,111 @@
+{{- $proxy_image := dict "image" "" }}
+{{- if .Values.Istio.proxy_debug }}
+{{- $_ := set $proxy_image "image" "proxy_debug" }}
+{{- else }}
+{{- $_ := set $proxy_image "image" "proxy" }}
+{{- end }}
+{{- $istio_namespace := .Values.Istio.namespace }}
+{{- $proxy_hub := .Values.Istio.proxy_hub }}
+{{- $proxy_tag := .Values.Istio.proxy_tag }}
+################################
+# Istio ingress
+################################
+apiVersion: v1
+kind: Service
+metadata:
+  name: istio-ingress
+  namespace: {{ $istio_namespace }}
+  labels:
+    istio: ingress
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+#   nodePort: 32000
```

Comment:
Why not make the choice between LoadBalancer and NodePort conditional?

Reply:
It is on todo list in the next revision of PR, at this point I am collecting feedback from the community on the taken approach.

Reply:
done